Product/Production/Services/PolicyEngineCore/src/main/java/gov/hhs/fha/nhinc/policyengine/adapter/pep/proxy/AdapterPEPProxyWebServiceUnsecuredImpl.java

```diff
@@ -86,13 +86,13 @@ public CheckPolicyResponseType checkPolicy(CheckPolicyRequestType request, Asser
                 CONNECTClient<AdapterPEPPortType> client = getCONNECTClientUnsecured(portDescriptor, url, assertion);
 
                 checkPolicyResponse = (CheckPolicyResponseType) client.invokePort(AdapterPEPPortType.class,
-                        "checkPolicy", request);
+                    "checkPolicy", request);
             } else {
                 LOG.error("Failed to call the web service (" + serviceName + ").  The URL is null.");
             }
         } catch (Exception ex) {
             String message = "Error occurred calling AdapterPEPProxyWebServiceUnsecuredImpl.checkPolicy.  Error: "
-                    + ex.getMessage();
+                + ex.getMessage();
```

Comment:
Optional: no need to log if we rethrow exception (https://sonarcloud.io/organizations/default/rules?languages=java&open=squid%3AS2139&q=log)

Reply:
deleted log.error